src/pages/FormPage/ErrorPage.tsx

```diff
@@ -0,0 +1,47 @@
+/** @jsx jsx */
+import {jsx, css} from "@emotion/react";
+import {Link} from "react-router-dom";
+import React from "react";
+
+import HeaderBar from "../../components/HeaderBar";
+import RenderedQuestion from "../../components/Question";
+
+import {Form} from "../../api/forms";
+import {selectable, submitStyles, unselectable} from "../../commonStyles";
+
+import Navigation from "./Navigation";
+
+
+interface ErrorProps {
+    form: Form
+    questions: RenderedQuestion[]
```

Comment:
Why is this included if it's never used?

Reply:
I initially included some serialization and deserialization for the refresh button, and just didn't get around to removing it. I'll do that

Reply:
b83dff71